config.go

```diff
@@ -386,6 +383,16 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
+	// At least one RPCListener is required
+	if len(cfg.RPCListeners) == 0 {
+		cfg.RPCListeners = append(cfg.RPCListeners, fmt.Sprintf("localhost:%d", defaultRPCPort))
```

Comment:
Can wrap this line to 80 chars either by moving the `Sprintf` into var above, or creating a new line for the second arg.